OpenCL.pc.in

```diff
@@ -0,0 +1,10 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+exec_prefix=${prefix}
+libdir=@OPENCL_LIBDIR_PC@
+includedir=@OPENCL_INCLUDEDIR_PC@
+
+Name: OpenCL
+Description: Khronos OpenCL ICD Loader
```

Comment:
```suggestion

Name: OpenCL
Description: Khronos OpenCL ICD Loader
Requires: OpenCL-Headers
```
This would work in conjunction with: https://github.com/KhronosGroup/OpenCL-Headers/pull/213 that would need to be merged before adopting this change.